dropwizard-testing/src/test/java/io/dropwizard/testing/junit5/DropwizardAppExtensionRandomPortsConfigOverrideTest.java

```diff
@@ -0,0 +1,28 @@
+package io.dropwizard.testing.junit5;
+
+import static io.dropwizard.testing.ConfigOverride.config;
+import static io.dropwizard.testing.ConfigOverride.randomPorts;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.dropwizard.testing.app.TestApplication;
+import io.dropwizard.testing.app.TestConfiguration;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@ExtendWith(DropwizardExtensionsSupport.class)
+class DropwizardAppExtensionRandomPortsConfigOverrideTest {
+
+    private static final DropwizardAppExtension<TestConfiguration> EXTENSION =
+        new DropwizardAppExtension<>(TestApplication.class,
+            null,
+            "app-rule",
+            randomPorts("app-rule"),
+            config("app-rule", "message", "A new way to say Hooray!"),
+            config("app-rule", "extra", () -> "supplied"));
+
+    @Test
+    void supportsConfigAttributeOverrides() {
+        assertThat(EXTENSION.getLocalPort()).isNotEqualTo(8080);
+        assertThat(EXTENSION.getAdminPort()).isNotEqualTo(8081);
```

Comment:
This is bound to be a flaky test if you run it often enough.

Shouldn't it suffice to check that both methods return an integer in the range of an IP port (0-65535)?

Reply:
I think it would also be better if I just check if the `Configuration` class has the expected content.

Reply:
I changed the test to not test the actual ports but just if the `HttpConnectorFactory`s were configured correctly.